Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -339,6 +339,36 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("closePath")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddAction("ClearBetweenFrames",
+                _("Clear between frames"),
+                _("Clear the renderer before the next draw"),
+                _("The shape painter _PARAM0_ is cleared before the next draw: _PARAM1_"),
+                _("Setup"),
+                "res/actions/visibilite24.png",
+                "res/actions/visibilite.png")
+
+      .AddParameter("object", _("Shape Painter object"), "Drawer")
+      .AddParameter("yesorno", _("Clear the previous draw"), "", true)
```

Comment:
"Clear between each frame?"

Reply:
No need "?", we have the checkbox for it :)